app/src/pages/playground/PlaygroundTemplate.tsx

```diff
@@ -45,55 +49,66 @@ export function PlaygroundTemplate(props: PlaygroundTemplateProps) {
   const { template } = instance;
 
   return (
-    <Card
-      title={
-        <Flex
-          direction="row"
-          gap="size-100"
-          alignItems="center"
-          marginEnd="size-100"
-        >
-          <AlphabeticIndexIcon index={index} />
-          <PromptComboBox
-            promptId={promptId}
-            onChange={(nextPromptId) => {
-              updateInstancePrompt({
-                instanceId,
-                patch: nextPromptId ? { id: nextPromptId } : null,
-              });
-            }}
-          />
-        </Flex>
-      }
-      collapsible
-      variant="compact"
-      bodyStyle={{ padding: 0 }}
-      extra={
-        <Flex direction="row" gap="size-100">
-          <Suspense
-            fallback={
-              <div>
-                <Loading size="S" />
-              </div>
-            }
+    <>
+      <Card
+        title={
+          <Flex
+            direction="row"
+            gap="size-100"
+            alignItems="center"
+            marginEnd="size-100"
           >
-            {/* As long as this component mounts, it will sync the supported
-            invocation parameters for the model to the instance in the store */}
-            <ModelSupportedParamsFetcher instanceId={instanceId} />
+            <AlphabeticIndexIcon index={index} />
+            <PromptComboBox
+              promptId={promptId}
+              onChange={(nextPromptId) => {
+                updateInstancePrompt({
+                  instanceId,
+                  patch: nextPromptId ? { id: nextPromptId } : null,
+                });
+              }}
+            />
+            <SaveButton instanceId={instanceId} setDialog={setDialog} />
```

Comment:
Let's put this on the right side of the model - the reason being that the model config also will get attached.

I think we maybe should start with just the "save" text rather than a floppy for it to be super clear